extras/CMakeLists.txt

```diff
@@ -24,6 +24,13 @@ target_link_libraries(largeSceneTest manifold)
 target_compile_options(largeSceneTest PRIVATE ${MANIFOLD_FLAGS})
 target_compile_features(largeSceneTest PUBLIC cxx_std_17)
 
+if(MANIFOLD_DEBUG)
```

Comment:
Since this is an extra, should we just go ahead and compile it always?

Reply:
no because it requires `MANIFOLD_DEBUG=on` so `geometryErr` is defined.